player.go

```diff
@@ -18,13 +19,48 @@ const (
 
 // Channel to detect when a track should be stopped
 var StopTrack chan struct{}
+var StopTimer chan struct{}
+
+type Ticker struct {
+	duration	int
+	step		int
+}
+
+func (t *Ticker) Start() {
+	t.step = 1
+	t.duration = 1
+}
+
+func (t *Ticker) Stop() {
+	t.duration = 1
+	t.step = 0
+}
+
+func (t *Ticker) Play() {
+	t.step = 1
+}
+
+func (t *Ticker) Increase() {
+	t.duration = t.duration + t.step
+}
+
+func (t *Ticker) Pause() {
+	t.step = 0
+}
+
+func NewTicker() *Ticker {
+	StopTimer = make(chan struct{}, 1)
+	return &Ticker{}
+}
+
 
 // Soundwave player, handles holding the connection to Spotify and
 // playing tracks
 type Player struct {
-	Audio   *audioWriter
-	Session *spotify.Session
-	Player  *spotify.Player
+	Audio   	 *audioWriter
+	Session 	 *spotify.Session
+	Player  	 *spotify.Player
+	TrackTicker  *Ticker
 }
```

Comment:
Can you install Go's formatting tools for Sublime, it will handle auto formatting the code to standards for you.